app/src/main/java/com/muzima/view/YouTubeVideoViewActivity.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2014 - 2018. The Trustees of Indiana University, Moi University
+ * and Vanderbilt University Medical Center.
+ *
+ * This version of the code is licensed under the MPL 2.0 Open Source license
+ * with additional health care disclaimer.
+ * If the user is an entity intending to commercialize any application that uses
+ *  this code in a for-profit venture,please contact the copyright holder.
+ */
+
+package com.muzima.view;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.widget.Toast;
+import com.google.android.youtube.player.*;
+import com.muzima.MuzimaApplication;
+import com.muzima.R;
+import com.muzima.service.TimeoutPreferenceService;
+
+import java.util.concurrent.TimeUnit;
+
+public class YouTubeVideoViewActivity extends BaseHelpActivity implements YouTubePlayer.OnInitializedListener {
+
+    private static String YOUTUBE_API_KEY = "AIzaSyB95WSRhfe-Pa6ZxU8ZB3C__E51ZQbZdu8";
+    private static final int RECOVERY_REQUEST = 1;
+    public static final String VIDEO_PATH = "VIDEO_PATH";
+    public static final String VIDEO_TITLE = "VIDEO_TITLE";
+    private String videoId;
+    private YouTubePlayer youTubePalyer;

```

Comment:
Should be youTubePlayer

Reply:
Hello @benamonya, I've done all the requested modifications. Thank you!